src/Dont/DeSerialize.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Dont;
+
+use Dont\Exception\NonDeSerializableObject;
+use Dont\Exception\TypeError;
+
+trait DeSerialize
```

Comment:
`Deserialise` is UK English btw, `Deserialize` is American, so technically, it's now Brexit-compliant :+1: :gb: